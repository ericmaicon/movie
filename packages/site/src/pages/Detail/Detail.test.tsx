import React from 'react';

import snapshotDiff from 'snapshot-diff';

import { render } from '~/test/index';

import { Detail } from './Detail';

const item = {
  id: 1,
  image:
    'https://www.themoviedb.org/t/p/w220_and_h330_face/drulhSX7P5TQlEMQZ3JoXKSDEfz.jpg',
  title:
    'When the crew of a space junk collector ship called The Victory discovers a humanoid..',
  description: '',
  rating: 1,
};

describe('Detail', () => {
  it('should render correctly', () => {
    const { getByTestId } = render(<Detail data-testid="detail" item={item} />);

    const detailElement = getByTestId('detail');

    expect(detailElement).toBeInTheDocument();
    expect(detailElement).toMatchSnapshot();
  });

  it('should render loading when there is no item', () => {
    const { rerender, asFragment } = render(
      <Detail data-testid="detail" item={null} />,
    );

    const loading = asFragment();
    rerender(<Detail data-testid="detail" item={item} />);
    const withItem = asFragment();

    expect(withItem).toMatchSnapshot();
    expect(snapshotDiff(loading, withItem)).toMatchSnapshot();
  });
});
