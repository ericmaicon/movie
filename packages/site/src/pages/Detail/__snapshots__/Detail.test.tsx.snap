// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail should render correctly 1`] = `
.c1 {
  margin: 0px;
  font-size: 1rem;
  font-family: Roboto;
  font-weight: 700;
  font-size: 4rem;
  line-height: 1em;
  -webkit-letter-spacing: 0em;
  -moz-letter-spacing: 0em;
  -ms-letter-spacing: 0em;
  letter-spacing: 0em;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c5 {
  cursor: pointer;
  margin: 0px 1px;
}

.c6 {
  cursor: pointer;
  margin: 0px 1px;
}

.c3 {
  font-size: 0.2rem;
  font-family: Roboto;
  font-weight: 300;
  font-size: 0.2rem;
  line-height: 1em;
  -webkit-letter-spacing: 0em;
  -moz-letter-spacing: 0em;
  -ms-letter-spacing: 0em;
  letter-spacing: 0em;
}

.c7 {
  cursor: pointer;
  width: 100%;
}

.c7:hover {
  box-shadow: 0px 0px 0px 2px rgba(0,0,0,0.2);
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  margin-bottom: 20px;
  margin-top: 50px;
}

.c2 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

@media (min-width:768px) and (max-width:1024px) {
  .c1 {
    font-size: 1.5rem;
  }
}

@media (min-width:1025px) {
  .c1 {
    font-size: 2rem;
  }
}

@media (min-width:768px) and (max-width:1024px) {
  .c3 {
    font-size: 0.5rem;
  }
}

@media (min-width:1025px) {
  .c3 {
    font-size: 1rem;
  }
}

<div
  data-testid="detail"
>
  <div
    style="box-sizing: border-box; position: relative; margin-left: auto; margin-right: auto; padding-left: 15px; padding-right: 15px; max-width: 960px;"
  >
    <div
      style="margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;"
    >
      <div
        class="c0"
        style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;"
      >
        <h4
          class="c1"
          size="4"
        >
          When the crew of a space junk collector ship called The Victory discovers a humanoid..
        </h4>
      </div>
      <div
        class="c2"
        style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;"
      >
        <p
          class="c3"
        >
          Rate:
        </p>
        <div
          class="c4"
        >
          <svg
            class="c5"
            data-testid="filled-star"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 1024 1024"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              voted-star
            </title>
            <path
              d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
            />
          </svg>
          <svg
            class="c6"
            data-testid="star"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 1024 1024"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              star
            </title>
            <path
              d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z"
            />
          </svg>
          <svg
            class="c6"
            data-testid="star"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 1024 1024"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              star
            </title>
            <path
              d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z"
            />
          </svg>
          <svg
            class="c6"
            data-testid="star"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 1024 1024"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              star
            </title>
            <path
              d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z"
            />
          </svg>
          <svg
            class="c6"
            data-testid="star"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 1024 1024"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              star
            </title>
            <path
              d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z"
            />
          </svg>
        </div>
      </div>
    </div>
    <div
      style="margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;"
    >
      <div
        style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;"
      >
        <img
          class="c7"
          src="https://www.themoviedb.org/t/p/w220_and_h330_face/drulhSX7P5TQlEMQZ3JoXKSDEfz.jpg"
        />
      </div>
      <div
        style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;"
      >
        <p
          class="c3"
        >
          <strong>
            Description:
          </strong>
           
          
        </p>
        <p
          class="c3"
        >
          <strong>
            Budget:
          </strong>
           
        </p>
        <p
          class="c3"
        >
          <strong>
            Popularity:
          </strong>
           
        </p>
        <p
          class="c3"
        >
          <strong>
            Release Date:
          </strong>
           
        </p>
        <p
          class="c3"
        >
          <strong>
            Revenue:
          </strong>
           
        </p>
        <p
          class="c3"
        >
          <strong>
            Tag Line:
          </strong>
           
        </p>
      </div>
    </div>
    <span
      style="display: table; clear: both;"
    />
  </div>
</div>
`;

exports[`Detail should render loading when there is no item 1`] = `
<DocumentFragment>
  .c1 {
  margin: 0px;
  font-size: 1rem;
  font-family: Roboto;
  font-weight: 700;
  font-size: 4rem;
  line-height: 1em;
  -webkit-letter-spacing: 0em;
  -moz-letter-spacing: 0em;
  -ms-letter-spacing: 0em;
  letter-spacing: 0em;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c5 {
  cursor: pointer;
  margin: 0px 1px;
}

.c6 {
  cursor: pointer;
  margin: 0px 1px;
}

.c3 {
  font-size: 0.2rem;
  font-family: Roboto;
  font-weight: 300;
  font-size: 0.2rem;
  line-height: 1em;
  -webkit-letter-spacing: 0em;
  -moz-letter-spacing: 0em;
  -ms-letter-spacing: 0em;
  letter-spacing: 0em;
}

.c7 {
  cursor: pointer;
  width: 100%;
}

.c7:hover {
  box-shadow: 0px 0px 0px 2px rgba(0,0,0,0.2);
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  margin-bottom: 20px;
  margin-top: 50px;
}

.c2 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

@media (min-width:768px) and (max-width:1024px) {
  .c1 {
    font-size: 1.5rem;
  }
}

@media (min-width:1025px) {
  .c1 {
    font-size: 2rem;
  }
}

@media (min-width:768px) and (max-width:1024px) {
  .c3 {
    font-size: 0.5rem;
  }
}

@media (min-width:1025px) {
  .c3 {
    font-size: 1rem;
  }
}

<div
    data-testid="detail"
  >
    <div
      style="box-sizing: border-box; position: relative; margin-left: auto; margin-right: auto; padding-left: 15px; padding-right: 15px; max-width: 960px;"
    >
      <div
        style="margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;"
      >
        <div
          class="c0"
          style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;"
        >
          <h4
            class="c1"
            size="4"
          >
            When the crew of a space junk collector ship called The Victory discovers a humanoid..
          </h4>
        </div>
        <div
          class="c2"
          style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;"
        >
          <p
            class="c3"
          >
            Rate:
          </p>
          <div
            class="c4"
          >
            <svg
              class="c5"
              data-testid="filled-star"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 1024 1024"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                voted-star
              </title>
              <path
                d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
              />
            </svg>
            <svg
              class="c6"
              data-testid="star"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 1024 1024"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                star
              </title>
              <path
                d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z"
              />
            </svg>
            <svg
              class="c6"
              data-testid="star"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 1024 1024"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                star
              </title>
              <path
                d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z"
              />
            </svg>
            <svg
              class="c6"
              data-testid="star"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 1024 1024"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                star
              </title>
              <path
                d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z"
              />
            </svg>
            <svg
              class="c6"
              data-testid="star"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 1024 1024"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                star
              </title>
              <path
                d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        style="margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;"
      >
        <div
          style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;"
        >
          <img
            class="c7"
            src="https://www.themoviedb.org/t/p/w220_and_h330_face/drulhSX7P5TQlEMQZ3JoXKSDEfz.jpg"
          />
        </div>
        <div
          style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;"
        >
          <p
            class="c3"
          >
            <strong>
              Description:
            </strong>
             
          </p>
          <p
            class="c3"
          >
            <strong>
              Budget:
            </strong>
             
          </p>
          <p
            class="c3"
          >
            <strong>
              Popularity:
            </strong>
             
          </p>
          <p
            class="c3"
          >
            <strong>
              Release Date:
            </strong>
             
          </p>
          <p
            class="c3"
          >
            <strong>
              Revenue:
            </strong>
             
          </p>
          <p
            class="c3"
          >
            <strong>
              Tag Line:
            </strong>
             
          </p>
        </div>
      </div>
      <span
        style="display: table; clear: both;"
      />
    </div>
  </div>
</DocumentFragment>
`;

exports[`Detail should render loading when there is no item 2`] = `
"Snapshot Diff:
- First value
+ Second value

  <DocumentFragment>
    <div
      data-testid=\\"detail\\"
    >
-     <span>
-       <span
-         class=\\"react-loading-skeleton css-1vmnjpn-skeletonStyles-Skeleton\\"
-         style=\\"height: 150px;\\"
+     <div
+       style=\\"box-sizing: border-box; position: relative; margin-left: auto; margin-right: auto; padding-left: 15px; padding-right: 15px; max-width: 960px;\\"
+     >
+       <div
+         style=\\"margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;\\"
+       >
+         <div
+           class=\\"sc-kstrdz bWkXzl\\"
+           style=\\"box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;\\"
+         >
+           <h4
+             class=\\"sc-eCssSg GDKwu\\"
+             size=\\"4\\"
+           >
+             When the crew of a space junk collector ship called The Victory discovers a humanoid..
+           </h4>
+         </div>
+         <div
+           class=\\"sc-hBEYos kCRGNi\\"
+           style=\\"box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;\\"
+         >
+           <p
+             class=\\"sc-iqHYGH buZNJQ\\"
+           >
+             Rate:
+           </p>
+           <div
+             class=\\"sc-pFZIQ nhztm\\"
            >
-         ‌
-       </span>
-     </span>
+             <svg
+               class=\\"sc-jrAGrp cYIEyM\\"
+               data-testid=\\"filled-star\\"
+               fill=\\"currentColor\\"
+               height=\\"1em\\"
+               stroke=\\"currentColor\\"
+               stroke-width=\\"0\\"
+               viewBox=\\"0 0 1024 1024\\"
+               width=\\"1em\\"
+               xmlns=\\"http://www.w3.org/2000/svg\\"
+             >
+               <title>
+                 voted-star
+               </title>
+               <path
+                 d=\\"M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z\\"
+               />
+             </svg>
+             <svg
+               class=\\"sc-kEjbxe iTrbxf\\"
+               data-testid=\\"star\\"
+               fill=\\"currentColor\\"
+               height=\\"1em\\"
+               stroke=\\"currentColor\\"
+               stroke-width=\\"0\\"
+               viewBox=\\"0 0 1024 1024\\"
+               width=\\"1em\\"
+               xmlns=\\"http://www.w3.org/2000/svg\\"
+             >
+               <title>
+                 star
+               </title>
+               <path
+                 d=\\"M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z\\"
+               />
+             </svg>
+             <svg
+               class=\\"sc-kEjbxe iTrbxf\\"
+               data-testid=\\"star\\"
+               fill=\\"currentColor\\"
+               height=\\"1em\\"
+               stroke=\\"currentColor\\"
+               stroke-width=\\"0\\"
+               viewBox=\\"0 0 1024 1024\\"
+               width=\\"1em\\"
+               xmlns=\\"http://www.w3.org/2000/svg\\"
+             >
+               <title>
+                 star
+               </title>
+               <path
+                 d=\\"M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z\\"
+               />
+             </svg>
+             <svg
+               class=\\"sc-kEjbxe iTrbxf\\"
+               data-testid=\\"star\\"
+               fill=\\"currentColor\\"
+               height=\\"1em\\"
+               stroke=\\"currentColor\\"
+               stroke-width=\\"0\\"
+               viewBox=\\"0 0 1024 1024\\"
+               width=\\"1em\\"
+               xmlns=\\"http://www.w3.org/2000/svg\\"
+             >
+               <title>
+                 star
+               </title>
+               <path
+                 d=\\"M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z\\"
+               />
+             </svg>
+             <svg
+               class=\\"sc-kEjbxe iTrbxf\\"
+               data-testid=\\"star\\"
+               fill=\\"currentColor\\"
+               height=\\"1em\\"
+               stroke=\\"currentColor\\"
+               stroke-width=\\"0\\"
+               viewBox=\\"0 0 1024 1024\\"
+               width=\\"1em\\"
+               xmlns=\\"http://www.w3.org/2000/svg\\"
+             >
+               <title>
+                 star
+               </title>
+               <path
+                 d=\\"M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3zM664.8 561.6l36.1 210.3L512 672.7 323.1 772l36.1-210.3-152.8-149L417.6 382 512 190.7 606.4 382l211.2 30.7-152.8 148.9z\\"
+               />
+             </svg>
+           </div>
+         </div>
+       </div>
+       <div
+         style=\\"margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;\\"
+       >
+         <div
+           style=\\"box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;\\"
+         >
+           <img
+             class=\\"sc-crrsfI bYspdc\\"
+             src=\\"https://www.themoviedb.org/t/p/w220_and_h330_face/drulhSX7P5TQlEMQZ3JoXKSDEfz.jpg\\"
+           />
+         </div>
+         <div
+           style=\\"box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 50%; flex-basis: 50%; flex-grow: 0; flex-shrink: 0; max-width: 50%; margin-left: 0%;\\"
+         >
+           <p
+             class=\\"sc-iqHYGH buZNJQ\\"
+           >
+             <strong>
+               Description:
+             </strong>
+              
+           </p>
+           <p
+             class=\\"sc-iqHYGH buZNJQ\\"
+           >
+             <strong>
+               Budget:
+             </strong>
+              
+           </p>
+           <p
+             class=\\"sc-iqHYGH buZNJQ\\"
+           >
+             <strong>
+               Popularity:
+             </strong>
+              
+           </p>
+           <p
+             class=\\"sc-iqHYGH buZNJQ\\"
+           >
+             <strong>
+               Release Date:
+             </strong>
+              
+           </p>
+           <p
+             class=\\"sc-iqHYGH buZNJQ\\"
+           >
+             <strong>
+               Revenue:
+             </strong>
+              
+           </p>
+           <p
+             class=\\"sc-iqHYGH buZNJQ\\"
+           >
+             <strong>
+               Tag Line:
+             </strong>
+              
+           </p>
+         </div>
+       </div>
+       <span
+         style=\\"display: table; clear: both;\\"
+       />
+     </div>
    </div>
  </DocumentFragment>"
`;
